src/types.ts

```diff
@@ -6,17 +6,32 @@ import type {
   ParameterSetDefinition,
 } from "./parameters/mod.ts";
 import type { ICustomType } from "./types/types.ts";
-
-// SlackManifestType is the top level type that imports all resources for the app
-// An app manifest is generated based on what this has defined in it
+// import type { CamelCasedPropertiesDeep } from "./types_utils/camel-case.ts";
```

Comment:
Why did you have to comment out this import again?

Reply:
I don't have to actually, it's just not in the use yet.